src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -43,4 +44,11 @@ public void signOutUser(String email) {
         
         return get(config.getParticipantsApi(offsetBy, pageSize), ACCOUNT_SUMMARY_PAGED_RESOURCE_LIST);
     }
+    
+    @Override
+    public StudyParticipant getStudyParticipant(String email) {
+        session.checkSignedIn();
+        
+        return get(config.getParticipant(email), StudyParticipant.class);
```

Comment:
Why did you use a constant in getAccountSummaries and StudyParticpant.class here? Preference/readability?